pkg/api-server/index_ws.go

```diff
@@ -1,18 +1,35 @@
 package api_server
 
 import (
-	"github.com/emicklei/go-restful"
+	"os"
+	"sync"
 
+	"github.com/emicklei/go-restful"
 	"github.com/Kong/kuma/pkg/api-server/types"
 	kuma_version "github.com/Kong/kuma/pkg/version"
 )
 
+var hostname string
+var once sync.Once
+
+func getHostname() string {
+	once.Do(func() {
+		name, err := os.Hostname()
+		if err != nil {
+			panic(err)
```

Comment:
We should just propagate error instead of throwing panic. Also instead of using `once.Do` we can retrieve it before creating a route. Like this:
```
func indexWs() (*restful.WebService, error) {
    hostname, err := os.Hostname()
    if err != nil {
        return nil, err
    }
	ws := new(restful.WebService)
	return ws.Route(ws.GET("/").To(func(req *restful.Request, resp *restful.Response) {
```

Reply:
Done.